R/metrics-sample.R

```diff
@@ -230,16 +245,87 @@ dss_sample <- function(observed, predicted, ...) {
 #' Forecasts with scoringRules, <https://www.jstatsoft.org/article/view/v090i12>
 #' @keywords metric
 
-crps_sample <- function(observed, predicted, ...) {
+crps_sample <- function(observed, predicted, separate_results = FALSE, ...) {
   assert_input_sample(observed, predicted)
 
-  scoringRules::crps_sample(
+  crps <- scoringRules::crps_sample(
     y = observed,
     dat = predicted,
     ...
   )
+
+  if (separate_results) {
+    medians <- apply(predicted, 1, median)
```

Comment:
We are adding this here to match the WIS implementation right? It seems like a bit of a shame given that it has no code overlap with the core crps code and so just makes this function more complicated/verbose/unclear? Or is it worth it for getting the output all at once?

Reply:
I agree but revisiting this would be for a separate issue potentially if it affects both this and the WIS implementation?